prov/shm/src/smr.h

```diff
@@ -145,6 +145,8 @@ struct smr_sar_entry {
 	size_t			iov_count;
 	struct ofi_mr		*mr[SMR_IOV_LIMIT];
 	bool			in_use;
+	struct ofi_mr_entry	*mr_entry;
```

Comment:
Since this is only used for the IPC path to get the handle from the IPC cache, let's rename to make that clear so it's not confusing with the above ofi_mr data.